fetch.bs

```diff
@@ -2640,6 +2640,11 @@ the request.
    <a for=request>origin</a> to  <var>request</var>'s
    <a for=request>client</a>'s <a for="environment settings object">origin</a>.
 
+   <li><p>Set <var>request</var>'s <a for=request>client-hints set</a> to be a <a
+   href="https://infra.spec.whatwg.org/#list-clone">clone</a> of the <a
```

Comment:
You can use `<a for=list>clone</a>`.

Reply:
Also, no newlines inside tags or inline elements (slightly different rules from HTML, sorry).

Reply:
cool